hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactSerializationTest.java

```diff
@@ -120,6 +114,21 @@ public void testSerializer_withDuplicateFieldNames() {
     }
 
     @Test
+    public void testReadWhenFieldDoesNotExist() {
+        SerializationConfig config = new SerializationConfig();
+        config.getCompactSerializationConfig()
+                .addSerializer(new NonExistingFieldReadingSerializer());
+
+        SerializationService service = createSerializationService(config);
```

Comment:
```suggestion
        SerializationService service = createSerializationService(NonExistingFieldReadingSerializer::new);
```
can be simplified to this